user/client/src/pages/Home/Home.css

```diff
@@ -117,12 +117,12 @@ section.scan h3 {
 
 section.about {
   background: #046364;
-  height: 100vh;
   display: flex;
```

Comment:
why did you remove the height?
not that it matters atm but it should he min-height so the section will hold all of the content in it

Reply:
removing the height made it responsive af